setup.py

```diff
@@ -310,6 +310,7 @@ def report(*args):
 # Version, create_version_file, and package_name
 ################################################################################
 package_name = os.getenv('TORCH_PACKAGE_NAME', 'torch')
+package_type = os.getenv('PACKAGE_TYPE', 'libtorch')
```

Comment:
Not sure why package_type is `libtorch`? When `setup.py` is invoked, most likely it is packaged as wheel, isn't it?
```suggestion
package_type = os.getenv('PACKAGE_TYPE', 'wheel')
```

Reply:
We have both PACKAGE_TYPE: wheel and PACKAGE_TYPE: libtorch